dashboard/app/models/unit_group.rb

```diff
@@ -312,24 +309,19 @@ def self.has_any_course_experiments?(user)
     Experiment.any_enabled?(user: user, experiment_names: UnitGroupUnit.experiments)
   end
 
-  # Get the set of valid courses for the dropdown in our sections table.
-  def self.valid_courses_without_cache
-    UnitGroup.all.select(&:launched?)
-  end
-
   # Returns whether the course id is valid, even if it is not "stable" yet.
   # @param course_id [String] id of the course we're checking the validity of
   # @return [Boolean] Whether this is a valid course ID
   def self.valid_course_id?(course_id, user = nil)
-    valid_courses(user: user).any? {|unit_group| unit_group.id == course_id.to_i}
+    UnitGroup.valid_courses(user: user).any? {|unit_group| unit_group.id == course_id.to_i}
```

Comment:
was this change intentional? `UnitGroup.` should be implied when called from another class method.

Reply:
I think I might have done it when valid_courses still wasn't working because I thought it might be the issue. I can put it back if needed